fixtures/js-stubs/authProvider.tsx

```diff
@@ -0,0 +1,10 @@
+import type {AuthProvider as AuthProviderType} from 'sentry/types';
```

Comment:
why do we have `OrganizationAuthProvider` and `AuthProvider`? 🧐

Reply:
Users can have SSO providers of their own (social login with github) and organizations can have SSO configuration for SAML/google domain/okta.

Reply:
Thanks @markstory, I will add that as the comment for anyone looking at this in the future :)